core/rfb.js

```diff
@@ -871,10 +876,48 @@ export default class RFB extends EventTargetMixin {
             return;
         }
 
-        if (this._viewOnly) { return; } // View only, skip mouse events
+        this._mousePos = { 'x': x, 'y': y };
+
+        // Limit many mouse move events to one every MOUSE_MOVE_DELAY ms
+        if (this._mouseMoveTimer == null) {
+
+            let timeToWait = MOUSE_MOVE_DELAY;
+
+            const timeSinceLastMove = Date.now() - this._mouseLastMoveTime;
+            if (timeSinceLastMove > MOUSE_MOVE_DELAY) {
+                this._sendMouse(x, y, this._mouseButtonMask);
+                this._mouseLastMoveTime = Date.now();
+            } else {
+                // Too soon since the latest move, wait the remaining time
+                timeToWait = MOUSE_MOVE_DELAY - timeSinceLastMove;
+            }
+
+            this._mouseMoveTimer = setTimeout(() => {
```

Comment:
Need to stop this on disconnect.

Reply:
Fixed

Reply:
Not really? You just tested that it didn't try to send anything. With the timer running we are still keeping references alive when not needed and possible other issues.

Reply:
I call clearTimeout() in _disconnect(), that should be enough.